modules/ROOT/pages/salesforce/salesforce-connector-config-topics.adoc

```diff
@@ -183,6 +183,9 @@ A Mule flow is divided in two. The first portion is usually an inbound endpoint
 
 Every time a subscription to `/u/TestStreaming` receives an event, it executes the rest of the flow. In the case of this example, it logs a message at INFO level.
 
+NOTE: The *Streaming Channel* field of the Subscribe Streaming Channel operation does not display change events that are available in the Salesforce environment because this information is not provided in the Salesforce API. However, your connector can subscribe to a streaming channel. For example, to subscribe to the `All Change Events` channel, use `/data/ChangeEvents` as the channel name to which you subscribe. See https://developer.salesforce.com/docs/atlas.en-us.change_data_capture.meta/change_data_capture/cdc_subscribe_channels.htm[Subscription Channels] in the Salesforce Change Data Capture Developer Guide.
```

Comment:
This is pretty long for a Note; do you suppose it should be part of the running narrative?

Also, I wonder about the placement of this information. This topic is titled Subscribe to a Streaming Channel, so is there a reason to say, after the rest of the topic text, that you can solve this problem by doing the very thing that the topic is about? Isn't this issue "covered" in the previous text?

These questions are for your consideration only; you need not respond to me directly.

One last question: would this note be just as effective if it comprised only the first and last sentences: the issue and the cross-ref?